docker/Dockerfile-docker-build

```diff
@@ -2,7 +2,8 @@ ARG docker_image
 FROM ${docker_image}
 
 RUN apt-get update \
- && apt-get install -y libsqlite3-dev postgresql-client
+ && apt-get install -y libsqlite3-dev postgresql-client libclang-4.0-dev \
+ && ln -s libclang-4.0.so.1 /usr/lib/x86_64-linux-gnu/libclang.so
```

Comment:
Why is this manual link step needed?

Reply:
For some reason, in the docker build, when you install `libclang-4.0-dev`, it doesn't actually create a `libclang.so` symlink (only `libclang-4.0.so` and `libclang-4.0.so.1` exist).  However, when crystal_lib runs, it expects to call the compiler with `-lclang`, and so it's looking for `libclang.so`.  Even `ldflags -p | grep clang` only shows `libclang-4.0`.  Googling around for why, most people seem to say "just symlink it".

*All* of the other Docker platforms install it correctly, and `libclang.so` is found just fine, so no need for symlinks.